spec/lib/facebook_data_analyzer/analyzeables/messages_spec.rb

```diff
@@ -0,0 +1,48 @@
+RSpec.describe FacebookDataAnalyzer::Messages do
+  subject do
+    described_class.new(catalog: test_catalog, options: {parallel: false} )
+  end
+
+  before(:each) do
+    subject.analyze
+  end
+
+  describe '#analyze' do
+    it 'should return proper @messages' do
+      message = subject.messages.first
+
+      expect(message.character_count).to eq(304)
```

Comment:
this test is failing with default example catalog. Also can't we setup Test Catalog to example by default for tests?